docs/doc/03-develop/02-nodejs.md

```diff
@@ -38,56 +38,56 @@ GRANT ALL on *.* TO user1;
 1. Copy and paste the following code to a file named `databend.js`:
 
 ```js title='databend.js'
-const mysql = require('mysql');
-const con = mysql.createConnection({
-   host: 'localhost',
-   port: 3307,
-   user: 'user1',
-   password: 'abc123',
-});
-
-con.connect((err) => {
-   if (err) throw err;
-   console.log('Connected to Databend Server!');
-
-   var sql = "CREATE DATABASE IF NOT EXISTS book_db";
-   con.query(sql, function (err, result) {
-      if (err) throw err;
-      console.log("Database created");
-   });
-
-   var sql = "USE book_db";
-   con.query(sql, function (err, result) {
-      if (err) throw err;
-   });
-
-
-   var sql = "CREATE TABLE IF NOT EXISTS books(title VARCHAR, author VARCHAR, date VARCHAR)";
-   con.query(sql, function (err, result) {
-      if (err) throw err;
-      console.log("Table created");
-   });
-
-   var sql = "INSERT INTO books VALUES('Readings in Database Systems', 'Michael Stonebraker', '2004')";
-   con.query(sql, function (err, result) {
-      if (err) throw err;
-      console.log("1 record inserted");
-   });
-
-   con.query("SELECT * FROM books", function (err, result, fields) {
-      if (err) throw err;
-      console.log(result);
-   });
-
-});
+const { Client } = require('databend-driver');
+
+const dsn = process.env.DATABEND_DSN
+    ? process.env.DATABEND_DSN
+    : "databend://user1:abc123@localhost:8000/default?sslmode=disable";
+
+function create_client() {
+    this.client = new Client(dsn);
```

Comment:
This was changed today, we could change it to latest v0.5.0.

Reply:
I will add this.conn later, thanks

Reply:
https://github.com/datafuselabs/databend/pull/12238/commits/92a478618690c6585d4681d768b480db4fd950bd